docs: describe how to manage embedded plugin registry for VS Code editor (#2539)
codenvy/che-docs: modules/administration-guide/pages/extensions-for-microsoft-visual-studio-code-open-source.adoc

:_content-type: ASSEMBLY
:description: Extensions for Microsoft Visual Studio Code - Open Source
:keywords: extensions, vs-code, vsx, open-vsx, marketplace
:navtitle: Extensions for Microsoft Visual Studio Code - Open Source
//:page-aliases:

[id="extensions-for-microsoft-visual-studio-code-open-source"]
= Extensions for Microsoft Visual Studio Code - Open Source

> To manage extensions, the link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source IDE] cloud editors use one of these link:https://open-vsx.org/about[Open VSX] registries:

Author: ```suggestion
To manage extensions, the IDE that loads in workspaces uses an link:https://open-vsx.org/about[Open VSX] registry instance:
```

Reviewer: why "a" ? there's only one code-rhel8 image per release per deployment.

Reviewer: "to manage exetensions, the editor uses an openvsx registry" is a lot cleaner IMHO.

Reviewer: "IDE that loads in workspaces" == "editor" :D 

why so wordy?

https://imgflip.com/i/77vsq9 :D

Reviewer: ```suggestion
To manage extensions, the editor uses an link:https://open-vsx.org/about[Open VSX] registry:
```

Author: :) Yes, but that's not typical for editors to load in workspaces. Combine that with the fact that users only visit the fewest docs page they can only when they really need something, I can't expect user guide readers to come from the same place as a tech lead on this project (of course they can learn it somewhere, which happens to be the docs, which is this file).

Reviewer: > not typical for editors to load in workspaces

ok now I'm confused. 

```
if 
   IDE = editor,
     and 
   workspace = filesystem w/ an editor (including extensions) and 1 or more checked out projects, 
then
  surely every workspace has an editor;
else 
  what would you use to edit files in the workspace? the terminal?
```

Do we have a "no editor" option when loading a workspace?

Author: On second thought, maybe I should add the VS Code name back into my suggestion because otherwise it's reads to general.

Author: ```suggestion
To manage extensions, the link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] IDE of the workspace uses an link:https://open-vsx.org/about[Open VSX] registry instance:
```

> To manage extensions, the link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] IDE of the workspace uses an link:https://open-vsx.org/about[Open VSX] registry instance:

Author: ```suggestion
To manage extensions, link:https://github.com/microsoft/vscode[this IDE] uses one of these link:https://open-vsx.org/about[Open VSX] registry instances:
```
